src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* `ResourceBuilder.CreateDefault` has detectors for
+  `OTEL_RESOURCE_ATTRIBUTES`, `OTEL_SERVICE_NAME` environment variables
+  so that explicit `AddEnvironmentVariableDetector` call is not needed. ([#2247](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2247))
```

Comment:
could you validate that we don't add resource attributes from env variable *twice* ?

Reply:
Test added https://github.com/open-telemetry/opentelemetry-dotnet/pull/2247/commits/ef5cb4b370a679906731cf4c093d75a8ac52e95d